src/Microsoft.ML.Transforms/RandomFourierFeaturizing.cs

```diff
@@ -607,19 +601,25 @@ private ValueGetter<VBuffer<float>> GetterFromFloatType(IRow input, int iinfo)
 
                 if (src.IsDense)
                 {
-                    featuresAligned.CopyFrom(src.Values, 0, src.Length);
+                    Array.Copy(src.Values, featuresAligned, src.Length);
```

Comment:
This is going to conflict with my change: #1650.  Instead it should be:

`src.GetValues().CopyTo(featuresAligned);`